### ISSUE TITLE
LiveMarks get not updated since 1.1 update

### ISSUE THREAD

**Issue by rcmcronny** [nt1m/livemarks#20]
I have not checked, if this worked with 1.0 version.

But now, the livemarks are not updated, the normal interval is 5 minutes, In the heise.de RSS Feed, there are 10 new items and the last item in the livemark is over 2h old.

Is there a way to debug this more, perhaps another addon is missbehaving or blocking ?

Thanks,
Ronny

**Comment by nt1m** [comment]
Hi, can you please open the Browser Console `Ctrl+Shift+J (or Cmd+Shift+J on a Mac)` and copy any errors you see ?
Thanks for reporting :)

**Comment by rcmcronny** [comment]
There are many, here are all from some minutes of the console open now :)

If you can give me a filter or keyword, i can try to narrow it down.

```
PushServiceWebSocket: _beginWSSetup: Not in shutdown state! Current state 2 PushServiceWebSocket.jsm:479
[Mitteilungsdetails anzeigen/ausblenden] Promise rejected after context unloaded: Promised response from onMessage listener went out of scope
toolbarContentscript.js:118
[Mitteilungsdetails anzeigen/ausblenden] Promise rejected after context unloaded: Promised response from onMessage listener went out of scope
toolbarContentscript.js:7
[Mitteilungsdetails anzeigen/ausblenden] Promise rejected after context unloaded: Promised response from onMessage listener went out of scope
toolbarContentscript.js:118
Error: Promised response from onMessage listener went out of scope
ExtensionMessagingService.js:89:9
Error: Promised response from onMessage listener went out of scope
ExtensionMessagingService.js:89:9
Error: Promised response from onMessage listener went out of scope ExtensionMessagingService.js:89:9
TypeError: (destructured parameter) is undefined background.js:30:1
Error: Promised response from onMessage listener went out of scope ExtensionMessagingService.js:89:9
Error: Promised response from onMessage listener went out of scope
ExtensionMessagingService.js:89:9
Error: Promised response from onMessage listener went out of scope
ExtensionMessagingService.js:89:9
NS_ERROR_DOM_BAD_URI: Component returned failure code: 0x805303f4 [nsIWebNavigation.loadURI] viewSource-content.js:287
Error: Invalid tab ID: 145
ext-browser.js
Error: Promised response from onMessage listener went out of scope ExtensionMessagingService.js:89:9
TypeError: (destructured parameter) is undefined background.js:30:1
Error: Promised response from onMessage listener went out of scope
ExtensionMessagingService.js:89:9
Error: Promised response from onMessage listener went out of scope
ExtensionMessagingService.js:89:9
[Mitteilungsdetails anzeigen/ausblenden] 1533645698569	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping 83b09962-b36b-445a-9315-8e9aad8ae706: {"name":"PingReadError","message":"","stack":"PingReadError@resource://gre/modules/TelemetryStorage.jsm:76:15\nloadPingFile@resource://gre/modules/TelemetryStorage.jsm:1675:13\n","becauseNoSuchFile":true} Log.jsm:853
Error: Promised response from onMessage listener went out of scope
ExtensionMessagingService.js:89:9
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
[Mitteilungsdetails anzeigen/ausblenden] popupDirection is not defined urlbarBindings_fx57.xml:263
[Mitteilungsdetails anzeigen/ausblenden] TypeError: this.input.maybeReplayDeferredKeyEvents is not a function[Weitere Informationen] urlbarBindings_fx57.xml:455:13
Error: Promised response from onMessage listener went out of scope ExtensionMessagingService.js:89:9
TypeError: (destructured parameter) is undefined background.js:30:1
Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf self blockiert ("script-src"). Source: call to eval() or related function blocked by CSP.
Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf self blockiert ("script-src"). Source: (function(a,v,q,P){var c={safeWindow:{},.... 20:1
Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf self blockiert ("script-src"). Source: call to eval() or related function blocked by CSP.
Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf self blockiert ("script-src"). Source: (function(a,v,q,P){var c={safeWindow:{},.... 20:1
Error: Promised response from onMessage listener went out of scope
ExtensionMessagingService.js:89:9
```

**Comment by nt1m** [comment]
That's useful enough, thanks!
